### ISSUE TITLE
: NVDA should be able to navigate between messages and it should not open the reactions menu when navigating with the errow keys.

### ISSUE THREAD

**Issue by FDX-VQ** [nvaccess/nvda#11860]

**Comment by CyrilleB79** [comment]
@FDX-VQ, I can see that you have opened various issues recently.
Most if not all of these issues lack a lot of information. If you want to get a change to have your requests looked at, you need to fill it more carefully.

For example in the current issue:
* you have used the new feature issue template but this is more likely a bug report
* you did even not tell in which application or on which website you have this issue. This cnanot be guessed!

When opening a new issue, please:
* use the correct template: bug report for an issue and feature request for a new feature.
* give all the required information for each paragraph (line beginning with # symbol)
* Do not write something on a line beginning by # symbol. Line beginning with # symbol are the questions and you should provide your answer on the line below.

In the existing issues:
Please add more details to the initial description by editing your first comment or by adding them in additional comments.

*

**Comment by leonardder** [comment]
I"m closing this one as per above comment.
